.github/workflows/main.yml

```diff
@@ -1018,12 +1079,6 @@ jobs:
         run: |
           mkdir -p install
           tar -xzvf ubuntu-gcc-install-ubuntu-24.04.tar.gz -C install
-      - name: Run perf count checker
```

Comment:
This seems to be removed by an accident

Reply:
No, perf count checker doesn't need because I create new form of perf measurament